go/saltpack/sign_test.go

```diff
@@ -506,19 +493,6 @@ func TestSignDetachedCorruptHeader(t *testing.T) {
 		t.Errorf("signer key %x, expected %x", skey.ToKID(), key.PublicKey().ToKID())
 	}
 
-	// remove the signature from the header
-	opts.corruptHeader = func(sh *SignatureHeader) {
-		sh.Signature = nil
-	}
-	sig, err = testTweakSignDetached(msg, key, opts)
-	if err != nil {
-		t.Fatal(err)
-	}
-	_, err = VerifyDetached(msg, sig, kr)
-	if err != ErrNoDetachedSignature {
-		t.Errorf("VerifyDetached error: %v, expected ErrNoDetachedSignature", err)
-	}
-
```

Comment:
Just to confirm: it's `verify.go` and `verify_stream.go` that we care about having 100%-ish-covered, right?